students/kimhyeji/join/views.py

```diff
@@ -0,0 +1,41 @@
+#from django.shortcuts import render
+import json
+from django.http import JsonResponse
+from django.views import View
+from .models import Join
+from django.db.models import Q
```

Comment:
여기도 파이썬 내장모듈, 장고 모듈, 그외 임포트문 순으로 정렬해주시고 import 문 기준 정렬해주세요!
```suggestion
import json

from django.http      import JsonResponse
from django.views     import View
from django.db.models import Q

from .models import Join
```